### PR TITLE
Doc update desktop readme

### PR DESCRIPTION
**Pull request by cameronperera**
# Description

I wanted to add the troubleshooting help for node-gyp issues while it was still fresh in my mind. I also fixed some of the linter issues in the README.

## Checklist

- [ ] I have added before and after screenshots for UI changes
- [ ] I have added JSON response output for API changes
- [ ] I have added steps to reproduce and test for bug fixes in the description
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: ui/desktop/README.md [hunk 1]
```diff
@@ -172,23 +174,29 @@ route using `await a11yAudit();`.
...
+We are aware of an issue of the desktop client not being able to start correctly when running as web app within development mode. This issue reproduces when the developer is enabling/disabling Mirage ([more info](#environment-variables-dev) about environment variables).
 
 The workaround to fix this issue is to reset/clear cookies and data (local storage and session storage).
+
+#### Node-gyp build errors after package upgrades
+
+If you are running into issues with running the desktop client after a node upgrade you may need to rebuild some tools needed for building the electron app. Be sure to remove `node_models/` for `ui/desktop/` and `ui/desktop/electron-app/` to make sure you have a clean slate. First thing you will need to check is if you have `python-setuptools` installed. It is needed for node-gyp to rebuild the native tools for electron. If you are on a mac and use Homebrew, you can run `brew install python-setuptools`. You should be able to run `yarn start:desktop` now. This will trigger the rebuild for electron but if you want to be safe you can run `./electron-app/node_modules/.bin/electron-rebuild` before trying to start up the desktop client.
```
Comment (ZedLi) [inline ui/desktop/README.md:198]: I would mention this is only for python v3.12 and above as well as the python command to install it if you don't use homebrew

Comment (cameronperera) [reply ui/desktop/README.md:198]: This wasn't an issue below python v3.12? Were those tools automatically included with those versions of python?

Comment (ZedLi) [reply ui/desktop/README.md:198]: Yup, exactly

Comment (cameronperera) [reply ui/desktop/README.md]: Updated. Let me know if that works.

**Review by ZedLi — APPROVED** [review]
Looks good, thanks for updating this!

**Review by calcaide — COMMENTED** [review]
Sorry I am late for this PR but thanks for updating this!! 🙌

**Pull request merged**